cernopendata/modules/fixtures/data/records/cms-trigger-information-2015.json

```diff
@@ -0,0 +1,31 @@
leUpFill/2015/ZeroTesla/v1.1/HLT/V4</a></td></tr>\n      <tr><td>247707 - 247710</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_2\">CMSSW_7_4_2</a></td><td><a href=\"/record/22531\">/cdaq/special/VeryLowPileUpFill/2015/ZeroTesla/v1.1/HLT/V5</a></td></tr>\n      <tr><td>247711 - 247720</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_2\">CMSSW_7_4_2</a></td><td><a href=\"/record/22532\">/cdaq/special/VeryLowPileUpFill/2015/ZeroTesla/v1.1/HLT/V6</a></td></tr>\n      <tr><td>247910 - 247915</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_4_patch4\">CMSSW_7_4_4_patch4</a></td><td><a href=\"/record/22524\">/cdaq/special/VeryLowPileUpFill/2015/Totem/HLT/V2</a></td></tr>\n      <tr><td>247917 - 247934</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_4_patch4\">CMSSW_7_4_4_patch4</a></td><td><a href=\"/record/22533\">/cdaq/special/VeryLowPileUpFill/2015/ZeroTesla/v1.2/HLT/V1</a></td></tr>\n      <tr><td>250930</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_6\">CMSSW_7_4_6</a></td><td><a href=\"/record/22514\">/cdaq/special/HLTPhysics/v1.0/HLT/V1</a></td></tr>\n      <tr><td>250932</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_6\">CMSSW_7_4_6</a></td><td><a href=\"/record/22516\">/cdaq/special/HLTPhysics/v1.0/HLT/V2</a></td></tr>\n      <tr><td>251023 - 251024</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_6\">CMSSW_7_4_6</a></td><td><a href=\"/record/22517\">/cdaq/special/HLTPhysics/v1.0/HLT/V7</a></td></tr>\n      <tr><td>251026 - 251027</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_6\">CMSSW_7_4_6</a></td><td><a href=\"/record/22518\">/cdaq/special/HLTPhysics/v1.0/HLT/V8</a></td></tr>\n      <tr><td>251250</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_6\">CMSSW_7_4_6</a></td><td><a href=\"/record/22515\">/cdaq/special/HLTPhysics/v1.0/HLT/V12</a></td></tr>\n      <tr><td>251717 - 251718</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_6\">CMSSW_7_4_6</a></td><td><a href=\"/record/22519\">/cdaq/special/Stage1Test/v1.1/HLT/V3</a></td></tr>\n      <tr><td>254980 - 254992</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_8_patch1\">CMSSW_7_4_8_patch1</a></td><td><a href=\"/record/22520\">/cdaq/special/VdMScan/v3.1/HLT/V19</a></td></tr>\n      <tr><td>257725 - 257734</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_12_patch4\">CMSSW_7_4_12_patch4</a></td><td><a href=\"/record/22521\">/cdaq/special/VdMScan/v3.2/HLT/V2</a></td></tr>\n      <tr><td>262081 - 262147</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_15\">CMSSW_7_4_15</a></td><td><a href=\"/record/22522\">/cdaq/special/VdMScan/v3.3/HLT/V1</a></td></tr>\n      <tr><td>262156 - 262164</td><td><a href=\"https://github.com/cms-sw/cmssw/tree/CMSSW_7_4_15\">CMSSW_7_4_15</a></td><td><a href=\"/record/22523\">/cdaq/special/VdMScan/v3.3/HLT/V2</a></td></tr>\n      </tbody>\n      </table>"
```

Comment:
this could be: "The list of trigger configuration files for the CMS 2015 proton-proton collision data (Run2015D from run number 256630 to run number 260627)."

i.e. 2015 instead of Run2015D to better indicate the fact that the config files that are listed are for all p-p data

Reply:
Changed to '2015'.